bundles/org.openhab.binding.urtsi/src/main/resources/OH-INF/i18n/urtsi_de.properties

```diff
@@ -10,9 +10,9 @@ urtsiDeviceCommandIntervalLabel = Befehlausführungsintervall
 urtsiDeviceCommandIntervalDescription = Die Zeit (in ms), die das Binding zwischen dem Absenden von zwei Befehlen warten soll
 
 rtsDeviceLabel = Somfy RTS Gerät
-rtsDeviceDescription = Dies ist ein Gerät, das über RTS kommuniziert (z.B. ein Rolladenmotor).
+rtsDeviceDescription =  Dies ist ein Gerät, das über RTS kommuniziert (z.B. ein Rolladenmotor).
```

Comment:
It looks like an unexpected leading space was added.
@cweitkamp : could you please fix that.

Reply:
Thanks for pointing this out. I had to fix the source file before I am able to change the translations in Crowdin. The space is in the source file too.

Reply:
See https://github.com/openhab/openhab-addons/pull/11757

Reply:
I propose to merge this PR before your new PR to avoid a ptobable new conflict there.